### PR TITLE
Cliu/ic path support doc

### PR DESCRIPTION
**Pull request by cliu0507**
CL:
1. add a new section in readme.md for ImageCutOut PS path support

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1359,6 +1364,29 @@ Once the job is complete your successful `/status` response will look similar to
 
 The workflow is exactly the same as [creating an image cutout](#example-1-initiate-a-job-to-create-an-image-cutout) except you use the `/mask` endpoint instead of `/cutout`.  
 
+## Customized Workflow
+This section will demonstrate how to make a 'customized workflow' by chaining different APIs. 
+
+### Example 3: (Generate ImageCutOut result as Photoshop path)
+This workflow is ONLY for users who'd like to generate cutout result as Photoshop path (instead of regular mask or cutout in above example 1 and example 2). You will need to chain API calls to ImageCutOut service and Photoshop Service to achieve this goal. 
+
```
Comment (khound) [inline README.md:1372]: Should we hyperlink to example 1 and example 2. Also a small correction, example 2 has an unwanted `)`

File: README.md [hunk 2]
```diff
@@ -1359,6 +1364,29 @@ Once the job is complete your successful `/status` response will look similar to
...
+#### Sample Input/Output
+Sample input from [here](assets/ic_customized_workflow/input.jpg)
```
Comment (khound) [inline README.md]: Since Sample input and Sample output are in the same line. Lets put `.` after first sentence.

**Comment by khound** [comment]
Looks good 👍

**Comment by cliu0507** [comment]
@khound Pushed new commits to fix the issue in comments.

**Pull request merged**